site/astro.config.ts

```diff
@@ -25,6 +25,15 @@ export default defineConfig({
   integrations: [
     starlight({
       title: "Zarf",
+      head: [
+        {
+          tag: "script",
+          content: "(function(w,d,s,l,i){ ... })(window,document,'script','dataLayer','G-N1XZ8ZXCWL');",
+        },
```

Comment:
this will need to be replaced w/ the actual contents of the google analytics script

Reply:
:facepalm: